src/chrome/index.js

```diff
@@ -13,7 +13,10 @@ import { applyMiddleware } from 'redux';
 import { logMiddleware } from '../logging';
 
 export const startChromePlugin = (chrome, socketConstructor = Socket) => {
-  const reloader = () => window.location.reload(true);
+  const reloader = () => {
+    chrome.storage.local.set({ reload: true });
```

Comment:
I think `set` is asynchronous, so you should put the reload call in the [`set` callback](https://developer.chrome.com/extensions/storage#type-StorageArea) to make sure that it completes.